purge_cluster.py

```diff
@@ -0,0 +1,79 @@
+"""Purge Ceph Cluster.
+
+Usage:
+    purge_cluster.py [--installer <IP>] [--username <USERNAME>] [--password <PASSWORD>]
+
+Options:
+    -h --help       Show this help message and exit.
+    --installer <IP>     Installer node IP.
+    --username <USERNAME>   SSH username.
+    --password <PASSWORD>   SSH password.
+
+Assumptions :
+    This will assume all the ceph nodes will have same root username password as provided for installer node
+"""
+
+import json
+
+import paramiko
+from docopt import docopt
+
+
+def execute_ssh_command(ssh_client, command):
+    stdin, stdout, stderr = ssh_client.exec_command(command)
+    return stdout.read().decode("utf-8")
+
+
+def get_fsid(ssh_client):
+    fsid_command = "cephadm shell -- ceph fsid -f json"
+    fsid_output = execute_ssh_command(ssh_client, fsid_command)
+    fsid_json = json.loads(fsid_output)
+    return fsid_json["fsid"]
+
+
+def get_node_ips(ssh_client):
+    node_ips_command = "cephadm shell -- ceph orch host ls -f json"
+    node_ips_output = execute_ssh_command(ssh_client, node_ips_command)
+    node_ips_json = json.loads(node_ips_output)
+    return [node["addr"] for node in node_ips_json]
+
+
+def main():
+    args = docopt(__doc__)
+
+    installer_node_ip = args["--installer"]
+    username = args["--username"]
+    password = args["--password"]
+
+    ssh_installer = paramiko.SSHClient()
+    ssh_installer.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh_installer.connect(installer_node_ip, username=username, password=password)
+
+    fsid = get_fsid(ssh_installer)
+
```

Comment:
As part of purging the cluster, we have to disable the cephadm. Please include the following step.
Command:  ceph mgr module disable cephadm

Reply:
I observed a problem when we disable cephadm,
we were not able to execute **cephadm rm-cluster** after disabling it.
Anyways we have added step tp uninstall cephadm